Symbol key1
Symbol key2
Pose relative_pose
NoiseModelPose noise
